template-transformer/handler.lua

```diff
@@ -133,15 +142,15 @@ function TemplateTransformerHandler:body_filter(config)
                                                                                            body = body,
                                                                                            raw_body = raw_body,
                                                                                            status = ngx.status}
-
+      ngx.log(ngx.ERR, string.format("BEFORE PREPARE :: %s", cjson_encode(transformed_body)))
```

Comment:
This is not an error, shouldn't this be an `INFO` ?

```suggestion
      ngx.log(ngx.INFO, string.format("BEFORE PREPARE :: %s", cjson_encode(transformed_body)))
```